### PR TITLE
Documenting the release process.

### PR DESCRIPTION
**Pull request by eddsaura**
## Subject

First proposal of release documentation.

### CODE REVIEW

File: docs/development/creating-a-release.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```batch
+npx auto-changelog --remote=[name_of_your_remote]
+```
+
+This command will output our updated `CHANGELOG.md`
+
+### Create new Release branch
+
+Now create the new branch to push all the changes made to the changelog. This way the team could review that everything is correct to approve it!
+
```
Comment (eddsaura) [inline docs/development/creating-a-release.md:39]:
```suggestion
Naming convention is release/[number_of_release], so if you are releasing the version 2.8.8 it should be `release/2.8.8`. 
```

File: docs/development/creating-a-release.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```batch
+npx auto-changelog --remote=[name_of_your_remote]
+```
+
+This command will output our updated `CHANGELOG.md`
+
+### Create new Release branch
+
+Now create the new branch to push all the changes made to the changelog. This way the team could review that everything is correct to approve it!
```
Comment (eddsaura) [inline docs/development/creating-a-release.md:38]:
```suggestion
Now create the new branch to push all the changes made to the changelog. This way the team could review that everything is correct to approve it!

```

File: docs/development/creating-a-release.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Naming convention is release/[number_of_release], so if you are releasing the version 2.8.8 it should be `release/2.8.8`.
```
Comment (eddsaura) [inline docs/development/creating-a-release.md:40]:
```suggestion
Naming convention is `release/[number_of_release]`, so if you are releasing the version 2.8.8 it should be `release/2.8.8`. 
```

**Comment by jordisala1991** [comment]
Make sure to link the page on the menu, otherwise it is not visible on the online docs.

Also notice that GitBook deploys automatically each PR, so you can check if your changes are okay before merge.

File: docs/development/creating-a-release.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+This command will output our updated `CHANGELOG.md`
+
+### Create new Release branch
+
+Now create the new branch to push all the changes made to the changelog. This way the team could review that everything is correct to approve it!
+
+Naming convention is `release/[number_of_release]`, so if you are releasing the version 2.8.8 it should be `release/2.8.8`. 
+### Create Github release
+
+With the approved branch, we head to [github releases](https://github.com/Stooa/Stooa/releases) to create a new one with the same number in the tag release we previously used in the `package.json`.
```
Comment (jordisala1991) [inline docs/development/creating-a-release.md:43]:
```suggestion
Once the PR is approved and merged, we head to [github releases](https://github.com/Stooa/Stooa/releases) to create a new one with the same number in the tag release we previously used in the `package.json`. 
```

File: docs/development/creating-a-release.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Naming convention is `release/[number_of_release]`, so if you are releasing the version 2.8.8 it should be `release/2.8.8`. 
+### Create Github release
+
+With the approved branch, we head to [github releases](https://github.com/Stooa/Stooa/releases) to create a new one with the same number in the tag release we previously used in the `package.json`. 
+
+The content of the release will be the last changes in the `CHANGELOG.md`
+
+### Release 🎉
+You can save a draft to further revision or just release it!
```
Comment (jordisala1991) [inline docs/development/creating-a-release.md:48]: I think I would not mention drafts for now. IMO they should be used previously to all those steps. But maybe is not something we need right now :)

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**